static/src/javascripts/projects/common/modules/spacefinder.js

```diff
@@ -218,27 +216,24 @@ const getReady = (rules, options) =>
 
 const getCandidates = (rules, exclusions) => {
 	let candidates = query(rules.bodySelector + rules.slotSelector);
+	let result;
 	if (rules.fromBottom) {
 		candidates.reverse();
 	}
 	if (rules.startAt) {
 		let drop = true;
-		exclusions.startAt = [];
-		candidates = filter(
-			candidates,
-			(candidate) => {
-				if (candidate === rules.startAt) {
-					drop = false;
-				}
-				return !drop;
-			},
-			exclusions.startAt,
-		);
+		result = filter(candidates, (candidate) => {
+			if (candidate === rules.startAt) {
+				drop = false;
+			}
+			return !drop;
+		});
+		exclusions.startAt = result.exclusions;
+		candidates = result.filtered;
 	}
 	if (rules.stopAt) {
 		let keep = true;
-		exclusions.stopAt = [];
-		candidates = filter(
+		result = filter(
```

Comment:
We can remove the third argument to filter, `exclusions.stopAt`.

Reply:
Oooh yeah nice catch